com.basemark.BasemarkGPU.yaml

```diff
@@ -0,0 +1,63 @@
+# Using electron example
+# https://github.com/flatpak/flatpak-builder-tools/blob/204309e0066a66a6f3c9ad7c5edb870513a7504c/node/vanilla-quick-start/org.electronjs.ElectronQuickStart.yaml
+# as a base/reference
+
+app-id: com.basemark.BasemarkGPU
+runtime: org.freedesktop.Platform
+runtime-version: "19.08"
+sdk: org.freedesktop.Sdk
+# Use the Electron 2 BaseApp, which adds several common libraries we'll need.
+base: org.electronjs.Electron2.BaseApp
+base-version: "19.08"
+# Electron doesn't use a traditional locale format so separate-locales is useless.
+separate-locales: false
+command: start-basemarkgpu
+rename-icon: basemarkgpu
+finish-args:
+  - "--socket=x11"
+  - "--share=ipc"
+  - "--share=network"
+  - "--device=dri"
+modules:
+  - dmidecode/dmidecode.json
+
+  - name: basemarkgpu
+    buildsystem: simple
+    build-commands:
+      - cp -ra basemarkgpu /app/basemarkgpu
+      - install -Dm755 start-basemarkgpu.sh /app/bin/start-basemarkgpu
+    sources:
+      - type: archive
+        url: https://cdn.downloads.basemark.com/BasemarkGPU-linux-1.2.0.tar.gz
+        # path: put local path here, comment url for testing
+        dest: basemarkgpu
+        sha256: 50f4031b028b4d403526e0be6a26cb5f83dd4c6dbd3bb274f7128aeac22df473
+      # For development/testing it's probably easier to just package the launcher directly from the electronui project
+      # comment out the archive source and use this directory source if you want to do that for faster iteration
+      # - type: dir
+      #   path: "../electronui/dist/linux-unpacked"
+      #   dest: basemarkgpu
+      # Wrapper script to start the launcher without sandbox
+      - type: script
+        dest-filename: start-basemarkgpu.sh
+        commands:
+          - "/app/basemarkgpu/basemarkgpu --no-sandbox"
+  - name: desktop
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 64x64/apps/basemarkgpu.png /app/share/icons/hicolor/64x64/apps/basemarkgpu.png
+      - install -Dm644 128x128/apps/basemarkgpu.png /app/share/icons/hicolor/128x128/apps/basemarkgpu.png
+      - install -Dm644 256x256/apps/basemarkgpu.png /app/share/icons/hicolor/256x256/apps/basemarkgpu.png
+      - install -Dm755 com.basemark.BasemarkGPU.desktop /app/share/applications/com.basemark.BasemarkGPU.desktop
+    sources:
+      - type: dir
+        path: icons
+      - type: file
+        path: com.basemark.BasemarkGPU.desktop
+  - name: metadata
+    buildsystem: simple
+    build-commands:
+      - install -Dm644 com.basemark.BasemarkGPU.appdata.xml -t /app/share/metainfo
+    sources:
+      - type: file
+        path: com.basemark.BasemarkGPU.appdata.xml
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html

Reply:
Should be resolved in 1aa95eaa943f350c1e33dc21730afab3d6f5338a